provider/datagen/src/tests/data.rs

```diff
@@ -21,7 +21,7 @@ impl DatagenProvider {
             .get_or_init(|| Self {
                 source: SourceData {
                     cldr_paths: Some(Arc::new(CldrCache::from_serde_cache(SerdeCache::new(AbstractFs::Memory(
```

Comment:
Why is there a diff in this file?

Reply:
there seems to be some instability in the gen script, which I ran locally. I'll fix in a separate PR